src/index.js

```diff
@@ -10,6 +10,7 @@ import sourceMap from 'source-map';
 const domPred = predicates.AND(predicates.hasTagName('dom-module'));
 const linkPred = predicates.AND(predicates.hasTagName('link'));
 const scriptsPred = predicates.AND(predicates.hasTagName('script'));
+const windowsPathRegExp = new RegExp('\\\\', 'g');
```

Comment:
I tend to prefer avoiding the `RegExp` constructor in most cases. `/\\\\/g` is equivalent here.